src/contexts/environment-context.js

```diff
@@ -37,6 +37,8 @@ export const EnvironmentProvider = ({ children }) => {
     <EnvironmentContext.Provider value={{
       helxSearchUrl: process.env.REACT_APP_HELX_SEARCH_URL,
       helxAppstoreUrl: window.location.origin,
+      enableSearch: process.env.REACT_APP_ENABLE_SEMANTIC_SEARCH,
```

Comment:
this is so minor, but having a verb as the first part of this variable makes me think that it might _do_ something, that it might be a function. i think maybe something like `searchEnabled` indicating that this is a thing that has been defined already, since it's in past tense. does that make sense?

Reply:
Yep, I think that sounds better! Will change those.